src/v/redpanda/admin/api-doc/debug.json

```diff
@@ -326,6 +360,79 @@
                     "description": "Recordings of test runs from a single node"
                 }
             }
+        },
+        "raft_state": {
+            "id": "raft_state",
+            "description": "Raft state",
+            "properties": {
+                "leader_info": {
+                    "type": "raft_leader_state",
+                    "description": "Leader info"
+                },
+                "followers": {
+                    "type": "array",
+                    "items": {
+                        "type": "raft_follower_state"
+                    },
+                    "description": "Raft followers info"
+                }
+            }
+        },
+        "raft_leader_state": {
+            "id": "raft_leader_state",
+            "description": "Raft leader state",
+            "properties": {
+                "id": {
+                    "type": "int",
+                    "description": "Node id"
+                },
+                "committed_log_index": {
```

Comment:
maybe we should consider returning the following offsets: 
```
_flushed_offset
_commit_index
_term
_majority_replicated_index
_visibility_upper_bound_index
_last_quorum_replicated_index
```
also last delta from offset translator
committed log index isn't really intersting as this is not the index that we use in raft. In raft we are using `commit_index`